user/user.go

```diff
@@ -17,56 +16,61 @@ import (
 )
 
 var (
-	validate         = validator.New()
-	EMAIL_NOT_VALID  = errors.New("Email address is not valid")
-	HASHING_FAILED   = errors.New("Failed to hashed password")
-	CONFIRM_PASSWORD = errors.New("Password and confirm password must be the same, confirm and try again!")
+	errEmailNotValid   = errors.New("email address is not valid")
+	errHashingFailed   = errors.New("failed to hashed password")
 )
 
-// Method to hash password
+// Method to hash password.
 func GenerateHashPassword(password string) (string, error) {
-	bytes, err := bcrypt.GenerateFromPassword([]byte(password), 14)
+	cost := 14
+	bytes, err := bcrypt.GenerateFromPassword([]byte(password), cost)
+
 	return string(bytes), err
 }
 
-// An end point to create new users
+// An end point to create new users.
 func (uh *UserHandler) Create(response http.ResponseWriter, request *http.Request) {
 	response.Header().Add("content-type", "application/json")
-	user_collection := "users"
+
+	userCollection := UserCollectionName
 
 	var user User
 	err := utils.ParseJsonFromRequest(request, &user)
+
 	if err != nil {
 		utils.GetError(err, http.StatusUnprocessableEntity, response)
 		return
 	}
 
 	userEmail := strings.ToLower(user.Email)
 	if !utils.IsValidEmail(userEmail) {
-		utils.GetError(EMAIL_NOT_VALID, http.StatusBadRequest, response)
+		utils.GetError(errEmailNotValid, http.StatusBadRequest, response)
 		return
 	}
 
 	// confirm if user_email exists
-	result, _ := utils.GetMongoDbDoc(user_collection, bson.M{"email": userEmail})
+	result, _ := utils.GetMongoDbDoc(userCollection, bson.M{"email": userEmail})
```

Comment:
result, _ := utils.GetMongoDbDoc(UserCollectionName, bson.M{"email": userEmail})

This should do. We need to limit the number of objects in memory especially when we do not need them. They seem very small but may have high impact when it scales.